### PR TITLE
feat(frontend): networks switcher

### PR DESCRIPTION
**Pull request by luzzif**

### CODE REVIEW

File: packages/frontend/src/components/connect-button/index.tsx [hunk 1]
```diff
@@ -24,47 +52,93 @@ export function ConnectButton() {
...
+                                    </Popover>
+                                    {!connected ? (
+                                        <Button
+                                            onClick={openConnectModal}
+                                            type="button"
+                                            className={{
+                                                root: styles.connectButton,
+                                            }}
+                                        >
+                                            Connect Wallet
```
Comment (guerrap) [inline packages/frontend/src/components/connect-button/index.tsx:118]: We should use the i18n keys here.

File: packages/frontend/src/components/connect-button/styles.module.css [hunk 2]
```diff
@@ -7,6 +7,26 @@
...
+.networkWrapper {
+    @apply bg-white w-10 h-10 rounded-md flex gap-3 items-center justify-center;
+}
+
+.networkIcon {
+    @apply w-7 h-7;
+}
+
+.networkRow {
+    @apply h-10 flex items-center px-2 gap-2 hover:bg-gray-200 transition-colors;
```
Comment (guerrap) [inline packages/frontend/src/components/connect-button/styles.module.css:22]: We should add the `duration-200` class here, for the transition.

File: packages/frontend/src/ui/popover/index.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            <div
+                ref={(element) => {
+                    if (ref) {
+                        if (typeof ref === "function") ref(element);
+                        else ref.current = element;
+                    }
+                    setPopper(element);
+                }}
+                style={{ ...popperStyles.popper }}
+                className={`${styles.root} ${className} ${open ? styles.open : ""}`}
```
Comment (guerrap) [inline packages/frontend/src/ui/popover/index.tsx:64]: Here is should be `className.root` no?

Comment (luzzif) [reply packages/frontend/src/ui/popover/index.tsx:64]: Nono `className` is right, but I needed to change the prop type to `string`.

File: packages/frontend/src/components/connect-button/styles.module.css (hunk 2 above)
Comment (luzzif) [reply packages/frontend/src/components/connect-button/styles.module.css]: Addressed in 72a15a7b69529c090da6887d86c5a5f2d3fb8b08

File: packages/frontend/src/ui/popover/index.tsx (hunk 3 above)
Comment (luzzif) [reply packages/frontend/src/ui/popover/index.tsx:64]: Addressed in 72a15a7b69529c090da6887d86c5a5f2d3fb8b08

**Review by guerrap — APPROVED** [review]

**Pull request merged**